dev/devicelab/bin/tasks/platform_services__build.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Each task defined in `bin/tasks` creates a new column in our [test matrix](https://flutter-dashboard.appspot.com/build.html). We therefore try to make our tests coarse grained. If we want to test multiple things, we bundle them in one task. For example, `flutter_gallery__transition_perf` tests all demo apps in the gallery app and measures performance all as part of one task. I intend to fix this at some point by supporting "task bundles". For now, I think we just need one task for platform services that tests everything we need. So let's remove `platform_services__build.dart` and do everything we need in the other one.